README.md

```diff
@@ -17,7 +17,7 @@ Feel free to browse, take what you like and contribute back if you want.
 | Communication | | |
 | --- | --- | --- |
 | [SerialCommunication sample pack](samples/SerialCommunication) | [CAN sample](samples/CAN) | [AMQP sample pack](samples/AMQP) |
-| [MQTT sample pack](samples/MQTT/TestMqtt) | [MQTT sample with Azure IoT Hub](samples/MQTT/AzureMQTT) | [MQTT sample with Amazon Web Services (AWS) IoT](samples/MQTT/AwsMQTT) |
+| [MQTT sample pack](samples/MQTT) | | |
```

Comment:
I would say that the explicit mention of both Azure IoT and AWS in the main readme increase the visibility of those samples and demonstrate that we support for both vendors.